### PR TITLE
add final retouches on my page

### PR DESCRIPTION
**Pull request by EddyShimwa**
### Highlights
- Checked linters✔
- Documented work in a professional way✔
- Styled my page ✔

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,7 +7,6 @@
     <title>LeaderBoard</title>
   </head>
   <body>
-    <h1>LeaderBoard</h1>
```
Comment (Abenezer-Tilahun) [inline src/index.html:10]: - [ ] As stated the project requirement Kindly  keep the general layout of the wireframe, this is the only mandatory requirement. So Kindly add "LeaderBoard " header Just like the wireframe.

Your Project Look likes this : 

Capture

The Leaderboard: Final Touches wireframe Look likes this  : 

Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @EddyShimwa  , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git-Flow.
✔️ Good README
✔️ Good commit messages

almost

## Required Changes ♻️

- [ ] Good Job for documenting your READEME professional way but I recommend you to add all that's needed for running the app with the API(s) it consumes. It should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them. Kindly Update it.

https://github.com/EddyShimwa/Leaderboard/blob/4923ca8827c97fcf664736c0f15f6d207a3e986b/README.md?plain=1#L88-L95

_Check the comments below the review._
## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @EddyShimwa 👋, 

You have done a great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by EddyShimwa** [comment]
Thanks youuuu!!!!!!!!!

**Pull request merged**